python class/task_4_class.py

```diff
@@ -0,0 +1,30 @@
+"""Успадкування
+Задача:
+
+Створіть клас Vehicle з атрибутами make, model і year
+Створіть підклас ElectricCar, який успадковується від Vehicle і додає атрибут battery_size (ціле число)
+Створіть екземпляр ElectricCar і виведіть атрибути як з Vehicle, так і з ElectricCar"""
+
+class Vehicle:
+    def __init__(self, make, model, year):
+        self.make = make
+        self.model = model
+        self.year = year
+
+    def get_info(self):
+        return f"Make: {self.make}\nModel: {self.model}\nYear: {self.year}" 
+
+# Дочірній клас
+class ElectricCar(Vehicle):
+    def __init__(self, make, model, year, battery_size):
+        # Виклик методу __init__ батьківського класу
+        Vehicle.__init__(self, make, model, year)
```

Comment:
Invoking parent classes in such way is kinda advanced and you should understand what you are trying to achieve.  In most cases just use `super()`. 
For more info visit: https://docs.python.org/2/library/functions.html#super
```suggestion
        super().__init__(make, model, year)
```